### PR TITLE
fix(accordion): animation for safari

### PR DESCRIPTION
**Pull request by ju-Skinner**
# Description

While investigating an issue reported, it was discovered that the animations were not working in Safari. It has been noted that transitions don't always work best in Safari. So I elected to use keyframes which seem to work well across all browsers. 

# Demo of before and after
https://www.loom.com/share/e32096c5bf4f4ae2b0f42f1748212eb5?sid=1244e615-466c-4a46-bcd2-a8b11bb88e80

# Notes
See the [slack message](https://kajabi.slack.com/archives/C03GKQJHCLX/p1714058690532659) about how to configure the local SSL Certs.

Fixes #(issue)
https://kajabi.atlassian.net/browse/DSS-582

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
Clicking the various panels to ensure they animate while opening.

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

### CODE REVIEW

File: libs/core/src/components/pds-accordion/pds-accordion.scss [hunk 1]
```diff
@@ -89,6 +97,7 @@ summary {
 }
 
 .pds-accordion__body {
+  animation: slide-down var(--animation-transform-timing);
```
Comment (pixelflips) [inline libs/core/src/components/pds-accordion/pds-accordion.scss:102]: Seeing 2 things and am not 100% sure why there's a difference.

1. Chrome doesn't seem to animate. Not sure if it's not animating or if 200ms is just super fast.

2. Safari now animates (and feels much slower than Chrome), but I see an overlap with the trigger and the sub-items.
Screenshot 2024-04-25 at 9 14 40 AM

If I move the `.pds-accordian__body` styles from line 99 into line 57 they both behave the same, but the overlap issue happens. This is in storybook, so not 100% sure if shows up in KP properly.

Comment (ju-Skinner) [reply libs/core/src/components/pds-accordion/pds-accordion.scss:102]: I added the original transitions back to the file. This seems to have fixed the Chrome issue you were referring to.

**Review by pixelflips — APPROVED** [review]

**Review by monicawheeler — APPROVED** [review]
Looks great! A small note, there are a couple of non-scss changes in this PR. Is that intentional or necessary? Would it be worth separating those into a separate PR? Thanks!

**Review by QuintonJason — APPROVED** [review]
Good to go

**Comment by ju-Skinner** [comment]
> Looks great! A small note, there are a couple of non-scss changes in this PR. Is that intentional or necessary? Would it be worth separating those into a separate PR? Thanks!

@monicawheeler We wouldn't be able to test the validity of these changes without the `non-scss` changes. So, no we should not separate them out. Great question though.

**Pull request merged**